common/religion/religions/water_group.txt

```diff
@@ -515,7 +515,7 @@
 			#Special
 			doctrine = special_doctrine_eastern_kingdoms_syncretism
 
-			doctrine = tenet_astrology
+			doctrine = tenet_arcane_scrying
```

Comment:
This really does not fit tidemother at all.
The reason as to why astrology was added in the first place was because of its naval bonus.
It should have its own tenet, like "Sacred Sailing", greatly boosting naval speed and other water-related modifiers.

Reply:
ok i'm open to this, although it might even be better to make it a unique cultural innovation or something so that jaina can keep it even being a kirin tor worshipper